tallycash/extension
# Fix activity item styling

Closes #2080 

This PR fixes the issue how the activity list item is looking when the token name is long.

**Changes**

When the name or amount is too long the name should jump to the next line. The address shouldn't jump to the next line.

**UI**

Before

<img width="240" alt="Screenshot 2022-09-06 at 20 21 07" src="https://user-images.githubusercontent.com/23117945/188804583-81ae6a05-a146-49b2-be7e-04fca7d64584.png">

After

<img width="240" alt="Screenshot 2022-09-06 at 20 21 07" src="https://user-images.githubusercontent.com/23117945/188804475-1cf1cfa2-156e-4a2f-86a5-800571d37c05.png">
<img width="240" alt="Screenshot 2022-09-06 at 20 20 30" src="https://user-images.githubusercontent.com/23117945/188804483-6fa7cd3c-b3e0-4841-9ed1-3ac488101b42.png">
<img width="240" alt="Screenshot 2022-09-06 at 20 19 48" src="https://user-images.githubusercontent.com/23117945/188804485-2a3ab17f-a63b-4d17-a457-a9c200a1c07d.png">



**To Test**

Go to the activity list and check how the activity list item looks.  Check a few variants: 
- [ ] When name is too long
- [ ] When amount is too long
- [ ] Short amount and short name
